backend/endpoints/rooms.py

```diff
@@ -23,7 +23,7 @@ async def extra_room_info(room_data: dict):
             "group_name": "channel",
             "room_name": room_data["room_name"],
         }
-    return {"category": "direct messagine", "group_name": "dm", "room_name": None}
+    return {"category": "direct messaging", "group_name": "dm", "room_name": None}
 
 
 @router.post("/org/{org_id}/members/{member_id}/room")
```

Comment:
KIndly add `response_model`, `status_code` and `responses`properties to the path operations. 

Here's an example: 
```
@router.put("/org/{org_id}/rooms/{room_id}/messages/{message_id}/read_status", 
            status_code=200, response_model=ReadStatus, 
            responses={404: {"model": MessageError}, 424: {"model": MessageError}})
```

Reply:
I have made these changes. Thank you

Reply:
Seen. Great.